### DISCUSSION TITLE
List of random findings

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#467] [Show and tell]
Chinese host

http://61.244.22.4/ch1/ch1.live/playlist.m3u8
http://61.244.22.4/ch2/ch2.live/playlist.m3u8
http://61.244.22.4/ch3/ch3.live/playlist.m3u8
http://61.244.22.4/ch4/ch4.live/playlist.m3u8
http://61.244.22.4/ch5/ch5.live/playlist.m3u8
http://61.244.22.4/ch6/ch6.live/playlist.m3u8

XXX KR 

rtmp://125.227.196.250:1935/live/Kchannel (unsure)

Other mix

http://seb.sason.top/sc/ztxw_fhd.m3u8 (中天新闻 - YouTube backed stream)

**Comment by PopeyeTheSai10r** [comment]
All but `http://61.244.22.4/ch1/ch1.live/playlist.m3u8` are down or Not 24/7.

**Reply by BellezaEmporium** [reply]
As of right now, channel 1,2 and 3 are working, the others are taking long to answer. The rest seems to have been cut indeed.
